Connect db survey

mos311063 opened the pull request:
add feature for user to able to be store current question and answer on survey

Commit:
fix section c bug(be able to next when not answer all question)

mmsunn committed:
add index.js for help testing

mmsunn committed:
Merge github.com:Magnify-Web-App/magnify-client into dynamic_survey

mmsunn committed:
merge and add user_session to survey

mmsunn committed:
save survey to db

mmsunn committed:
merge fix

mmsunn committed:
fix section C and D not render properly

sukibeww reviewed (approved):
massive refactoring work! awesome moss 😀

sukibeww reviewed (commented):
What are the purposes of this index.js(s) ??

sukibeww committed:
Styled save progress feature

added a floating action button and a snackbar for user feedback using material UI library.

mos311063 committed:
Merge pull request #8 from sukibeww/landing

Styled save progress feature
